### DISCUSSION TITLE
[Support] 将httpclient.d.ts中的IOptions也进行导出！

### DISCUSSION THREAD

**Discussion by minefujiko9293** [midwayjs/midway#3476] [Q&A] [answered]
### Describe the problem（描述问题）

可以将httpclient中的 IOptions 也进行导出吗？有时候封装方法时要用到。
![image](https://github.com/midwayjs/midway/assets/23348616/3985dbe0-7457-44f6-89bf-86c7099cf5c9)

### Midway Versions（Midway 版本）

>> Start to check your midway component version...

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed) 
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.12.3)
✓ @midwayjs/axios(not installed)      
✓ @midwayjs/bootstrap(3.12.5)
✓ @midwayjs/bull(3.12.3)
✓ @midwayjs/bull-board(3.12.7)    
✓ @midwayjs/cache(not installed)  
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed) 
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)   
✓ @midwayjs/consul(not installed)     
✓ @midwayjs/core(3.12.3)
✓ @midwayjs/cos(not installed)        
✓ @midwayjs/cron(not installed)       
✓ @midwayjs/cross-domain(3.12.7)      
✓ @midwayjs/decorator(3.12.3)
✓ @midwayjs/etcd(not installed)       
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)       
✓ @midwayjs/grpc(not installed)       
✓ @midwayjs/http-proxy(not installed) 
✓ @midwayjs/i18n(3.12.10)
✓ @midwayjs/info(3.12.7)
✓ @midwayjs/jwt(3.12.7)
✓ @midwayjs/kafka(not installed)   
✓ @midwayjs/leoric(not installed)  
✓ @midwayjs/mikro(not installed)   
✓ @midwayjs/mock(3.12.3)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/oss(not installed)     
✓ @midwayjs/otel(not installed)    
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed) 
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)   
✓ @midwayjs/redis(3.12.3)
✓ @midwayjs/security(not installed)   
✓ @midwayjs/sequelize(not installed)  
✓ @midwayjs/session(3.12.3)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)✓ @midwayjs/swagger(not installed)    
✓ @midwayjs/tablestore(not installed) 
✓ @midwayjs/tags(not installed)       
✓ @midwayjs/typegoose(not installed)  
✓ @midwayjs/typeorm(3.12.3)
✓ @midwayjs/upload(not installed)
✓ @midwayjs/validate(3.12.7)
✓ @midwayjs/version(3.12.7)
✓ @midwayjs/view(not installed)       
✓ @midwayjs/view-ejs(not installed)   
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)        
✓ @midwayjs/express(not installed)    
✓ @midwayjs/koa(3.12.3)
✓ @midwayjs/ws(not installed)
**********************************************************************      
>> Check complete, all versions are healthy.
**********************************************************************

**Comment by minefujiko9293** [comment]
@czy88840616

**Reply by czy88840616** [reply]
可以的

**Reply by czy88840616** [reply, marked as answer]
#3478

**Comment by czy88840616** [comment]
已经发布 3.13.5，导出了 HttpClientOptions
